packages/cli/src/databases/migrations/mysqldb/1666304975928-CreateLdapEntities.ts

```diff
@@ -0,0 +1,56 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { LDAP_DEFAULT_CONFIGURATION } from '@/Ldap/constants';
+import { getTablePrefix, logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+
+export class CreateLdapEntities1666304975928 implements MigrationInterface {
+	name = 'CreateLdapEntities1666304975928';
+
+	async up(queryRunner: QueryRunner): Promise<void> {
+		logMigrationStart(this.name);
+		const tablePrefix = getTablePrefix();
+
+		await queryRunner.query(
+			`ALTER TABLE \`${tablePrefix}user\`
+			ADD COLUMN \`ldapId\` varchar(60) DEFAULT null,
+			ADD COLUMN \`signInType\` varchar(20) DEFAULT \'email'\,
+			ADD COLUMN \`disabled\` boolean NOT NULL DEFAULT false;`,
+		);
+
+		await queryRunner.query(
+			`CREATE UNIQUE INDEX \`UQ_${tablePrefix}1b8332f7a44b48c3805d43db5627542b\` ON \`${tablePrefix}user\` (\`ldapId\`)`,
+		);
+
+		await queryRunner.query(
+			`CREATE TABLE IF NOT EXISTS feature_config (
```

Comment:
`tablePrefix` is missing here

Reply:
closing as this got fixed with the feature/ad-ldap-suggestions